ITEM_en.tsv

```diff
@@ -1447,9 +1446,9 @@ ITEM_20150317_001548	Shepherd Costume
 ITEM_20150317_001549	Chaplain Costume
 ITEM_20150317_001550	Empty_Hat
 ITEM_20150317_001551	Hair Costume
-ITEM_20150317_001552	Costume for the hair slot. You can equip it by right-clicking the item.
+ITEM_20150317_001552	A Costume for the Hat slot. You can equip it by right-clicking the item.
```

Comment:
*Hair decoration. Right-click to equip.* 

Hat is incorrect. The first sentence refers to the item being decoration for the player's hairstyle.